packages/terra-form-textarea/src/Textarea.jsx

```diff
@@ -210,17 +212,22 @@ class Textarea extends React.Component {
       ...customProps
     } = this.props;
 
+    const theme = this.context;
```

Comment:
shouldn't these be React.useContext(ThemeContext); ...?

Reply:
For the class Component we can use `class.contextType` to set ThemeContext to variable `this.context`.

You can check it out over here https://reactjs.org/docs/context.html#classcontexttype